VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -479,4 +480,12 @@ export class ParticipantWaitingRoomComponent implements OnInit, OnDestroy {
             return 'hearing-delayed';
         }
     }
+
+    muteUnmuteCall() {
+        if (this.pexipAPI) {
+            var muteAudio = this.pexipAPI.muteAudio();
```

Comment:
muteAudio returns a true to false to confirm the current status. Use that to update the audioMuted property